tests/catch/unit/stream/CMakeLists.txt

```diff
@@ -25,9 +25,13 @@ set(TEST_SRC
     hipStreamCreateWithFlags.cc
     hipStreamCreateWithPriority.cc
     hipAPIStreamDisable.cc
+    # hipStreamAttachMemAsync.cc   # Disabling it on nvidia due to issue in function definition of hipStreamAttachMemAsync
```

Comment:
@gandryey Any comment on this?

Reply:
hipStreamAttachMemAsync is a nop for AMD. I'm not entirely sure what should be done there. We always have global attachment and transfer data per a GPU page fault. It seems nvidia had some kind of SW management before and that call could optimize data transfers per stream.